config/ConfigExample/RemoteConfigViewController.swift

```diff
@@ -196,7 +209,7 @@ extension UIViewController {
   public func displayError(_ error: Error?, from function: StaticString = #function) {
```

Comment:
I think the error you were running into had to do with how this method was being called within the async contexts. It was then being called on a background thread causing the crash. Weird how it wasn't straightforward to reproduce though. I think it would work if the implementation of this method was wrapped in `DispatchQueue.main.async { ... }`. That way, you could call it from whatever context and it would put itself on the main queue by itself.

Reply:
Thanks Nick!  If you have the bandwidth, would you see if you can get the async/await variation to work?  Otherwise I should be able to get it to work later this week.